packages/core/lib/console-child.js

```diff
@@ -0,0 +1,51 @@
+const Command = require("../lib/command");
+const TruffleError = require("@truffle/error");
+const Config = require("@truffle/config");
+const Web3 = require("web3");
+
+const inputStrings = process.argv[2];
+//deserialize options
+const options = JSON.parse(process.argv[3]);
+
+//deserialize networks entries
+for (var key in options.networks) {
```

Comment:
Can we use `const key`?

Reply:
I removed this part of the code, so no need! 🎉